Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'valid_email'
 gem 'rack-attack'
 gem 'rqrcode'
 gem 'ruby-saml'
-gem 'saml_idp', '~> 0.3.1'
+gem 'saml_idp', '~> 0.3.1', github: 'pkarman/saml_idp', branch: 'principal-asserted-attributes'
```

Comment:
I'd like to propose that when we pin to local versions like this we include a comment as to why, and a reference to a remind us that we need to push upstream.  What do you guys think?